Landing page/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="header">
+      Microverse Supermarket
+      <img
+        src="https://i.pinimg.com/600x315/76/1b/a3/761ba3e1ddf1e70f4cb06906d5935668.jpg"
+        id="header-img"
+        alt="A supermarket clip art"
+      />
+      <br />
+      <br />
+      <br />
+      <br />
```

Comment:
Consider using box model's, margin or padding or use flex-box or grid instead of using break tags. This will make your code more professional and will add an aesthetic feel to your page as well as better arrangement.